src/main/java/mate/academy/bookstore/controller/BookController.java

```diff
@@ -0,0 +1,56 @@
+package mate.academy.bookstore.controller;
+
+import java.util.List;
+import mate.academy.bookstore.dto.BookDto;
+import mate.academy.bookstore.dto.BookSearchParametersDto;
+import mate.academy.bookstore.dto.CreateBookRequestDto;
+import mate.academy.bookstore.service.BookService;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/books")
+public class BookController {
+    private final BookService bookService;
+
+    public BookController(BookService bookService) {
+        this.bookService = bookService;
+    }
+
+    @PostMapping
+    public BookDto save(@RequestBody CreateBookRequestDto dto) {
```

Comment:
```suggestion
    public BookDto save(@RequestBody CreateBookRequestDto createBookRequest) {
```
let's change it to be more informative and readable